### ISSUE TITLE
[Request] Allow date input on Dialog

### ISSUE THREAD

**Issue by zuck** [quasarframework/quasar#1040]
Add support for `QDatetime` and `QDatetimeRange` to Dialog's forms.

```
Dialog.create({
  // ...
  form: {
    birthdate: {
      type: 'datetime', // 'datetime-range'
      model: ...
    }
  }
})
```

**Comment by HenriqueSPin** [comment]
+1

**Comment by rstoenescu** [comment]
Available in v0.15 through QDialog component.
